client/planning-extension/src/extension.ts

```diff
@@ -123,12 +123,12 @@ const extension: IExtension = {
                         getActions: (item) => [
                             {
                                 label: gettext('Unlink as Coverage'),
-                                groupId: 'planning-actions',
+                                groupId: planningActionsGroupId,
                                 icon: 'cut',
                                 onTrigger: () => {
                                     const superdeskArticle = superdesk.entities.article;
 
-                                    // keep in sync with client/planning-extension/src/extension.ts:123
+                                    // keep in sync with index.ts:108
```

Comment:
better keep relative path. There are lots of index files

Reply:
If you mean the absolute it would be something like superdesk-planning/index.ts

If relative, then this is it, and coincidentally index.ts really is the path & file name, relative to the project.

Reply:
I meant relative to the project. You get it by right clicking open file tab and choosing "copy relative path"